src/Sulu/Bundle/LocationBundle/Geolocator/Service/NominatimGeolocator.php

```diff
@@ -39,16 +40,19 @@ class NominatimGeolocator implements GeolocatorInterface
      */
     private $key;
 
-    public function __construct(ClientInterface $client, string $baseUrl, string $key)
-    {
-        $this->client = $client;
+    public function __construct(
+        HttpClientInterface $httpClient,
```

Comment:
This is a BC break. We need here handle that somebody maybe inject the the guzzle client.